README.md

```diff
@@ -40,15 +32,19 @@ Release log
 - Param option for boot sound
 - Param for no boot sound
 
+##### 1.2.1
+- GPIO Pin command line option
+- button type NO and NC
```

Comment:
This is controlled via command line option flag. I should update the doc to reflect it.